modules/csg/editor/csg_gizmos.cpp

```diff
@@ -541,7 +542,7 @@ EditorPluginCSG::EditorPluginCSG() {
 	Node3DEditor::get_singleton()->add_gizmo_plugin(gizmo_plugin);
 
 	csg_shape_editor = memnew(CSGShapeEditor);
-	EditorNode::get_singleton()->get_main_screen_control()->add_child(csg_shape_editor);
+	EditorNode::get_singleton()->get_editor_main_screen()->get_control()->add_child(csg_shape_editor);
```

Comment:
Editor's that don't have their screen should use `get_gui_base()` instead of main screen control.